src/script/standard.h

```diff
@@ -80,9 +80,15 @@ struct PKHash : public uint160
     using uint160::uint160;
 };
 
+struct WitnessV0KeyHash;
+struct WitnessV0ScriptHash;
 struct ScriptHash : public uint160
 {
     ScriptHash() : uint160() {}
+    // These don't do what you'd expect.
+    // Use ScriptHash(GetScriptForDestination(...)) instead.
+    explicit ScriptHash(const WitnessV0KeyHash& hash) = delete;
+    explicit ScriptHash(const WitnessV0ScriptHash& hash) = delete;
```

Comment:
Not sure the second one is needed... WitnessV0ScriptHash is a uint256, which has no constructor?

Reply:
fixed, added `PKHash` instead since that *is* possible